bin/varnishtest/tests/d00007.vtc

```diff
@@ -84,7 +116,9 @@ client c1 {
 	txreq -url /1
 	rxresp
 	expect resp.status == 200
+	expect resp.http.Authority == <undef>
```

Comment:
Somehow the authority manages to be `<undef>` here, I should probably understand why.

Reply:
we simply go straight to s1, no via.